components/insight/SRC/org/openmicroscopy/shoola/agents/treeviewer/ExperimenterDataLoader.java

```diff
@@ -240,7 +240,7 @@ public void handleResult(Object result)
     		Class klass = parent.getUserObject().getClass();
     		long id = parent.getUserObjectId();
     		if (TagAnnotationData.class.equals(klass)) {
-    			viewer.setLeaves((Set) result, parent, expNode);
+    			viewer.setLeaves((Collection) result, parent, expNode);
```

Comment:
There's a few bare collections around. I assume that there's nothing that makes sense in the `<>`, even `?`? (I can imagine issues might just propagate too far for now.)